docs/excel/custom-functions-batching.md

```diff
@@ -27,60 +27,56 @@ To set up batching for your custom functions you'll need to write three main sec
 2. A function to make the remote request when the batch is ready.
```

Comment:
I agree with Linda's suggestion of adding inline links to each of these three steps.